script/xwd2png.py

```
@@ -0,0 +1,84 @@
+            p = row_start + x * bpp
+            chunk = pixels[p:p + bpp]
+            if len(chunk) < bpp:
+                px = 0
+            elif byte_order == 0:  # LSBFirst
+                px = int.from_bytes(chunk, "little")
+            else:
+                px = int.from_bytes(chunk, "big")
+            r = (px & red_mask) >> rsh
+            g = (px & green_mask) >> gsh
+            b = (px & blue_mask) >> bsh
+            if rw and rw < 8: r <<= (8 - rw)
```

Ruff (0.15.14)

[error] Multiple statements on one line (colon)

(E701)